docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,92 @@
+export SERVICE_NAME=catalog-catalog-apiserver
+export ALT_NAMES="\"${SERVICE_NAME}.${NAME_SPACE}\",\"${SERVICE_NAME}.${NAME_SPACE}.svc"\"
+echo '{"CN":"'${SERVICE_NAME}'","hosts":['${ALT_NAMES}'],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -ca=${SERVINGCA_CERT} -ca-key=${SERVINGCA_KEY} -config=server-ca-config.json - | cfssljson -bare apiserver
+```
+# put the output we need for running our apiserver behind the aggregator
+
+Create some variables to use later that hold the names of the resulting keys.
+```
+export SC_SERVING_CERT=apiserver.pem
+export SC_SERVING_KEY=apiserver-key.pem
+```
+
+
+# TODO if rbac is enabled, allow the service catalog to grant access for it's service token.
+Need to figure out all of the permissions we need, and group them into a role.
+
+# not sure if I still need to create this (incorrectly spelled) namespace
+kubectl create namespace catlog
+# figure out RBAC rules, these are only some
+for the error
+```
+Unable to get configmap/extension-apiserver-authentication in kube-system.  Usually fixed by 'kubectl create rolebinding -n kube-system ROLE_NAME --role=extension-apiserver-authentication-reader --serviceaccount=YOUR_NS:YOUR_SA'
+```
+run
+```
+kubectl create rolebinding -n kube-system read-kube-system-configmap --role=extension-apiserver-authentication-reader --serviceaccount=catalog:default
+```
+# logging error output: 
+for
+```
+"Internal Server Error: \"/healthz\": User \"system:serviceaccount:catalog:default\" cannot create subjectaccessreviews.authorization.k8s.io at the cluster scope. (post subjectaccessreviews.authorization.k8s.io)"
+```
+run
+```
+kubectl create rolebinding -n kube-system catalogkubeauthdelegation --role=system:auth-delegator --serviceaccount=catalog:default
+```
+
+more stuff
+```
+what I did do
+  ca kubectl create rolebinding -n kube-system catalogkubeauthdelegation --role=system:auth-delegator --serviceaccount=catalog:default
+rolebinding "catalogkubeauthdelegation" created
+  ca kubectl create clusterrolebinding service-catalog:system:auth-delegator --clusterrole=system:auth-delegator --serviceaccount=catalog:default
+clusterrolebinding "service-catalog:system:auth-delegator" created
+```
+
+
+
+# install the thing
+
+enabling the auth and aggregator along with providing the correct keys inline.
+
+```
+helm install /root/go/src/github.com/kubernetes-incubator/service-catalog/charts/catalog --name catalog --namespace catalog \
+    --set apiserver.auth.enabled=true \
+        --set useAggregator=true \
+        --set apiserver.tls.ca=$(base64 --wrap 0 ${SERVINGCA_CERT}) \
+        --set apiserver.tls.cert=$(base64 --wrap 0 ${SC_SERVING_CERT}) \
+        --set apiserver.tls.key=$(base64 --wrap 0 ${SC_SERVING_KEY})
+``` 
+
+You can see we use the generated cert and key, along with the cores CA
+certificate. (TODO: Is this good or bad practice? Is there an
+alternative delegated kind of CA we can use if we don't have access to
+kube itself on a cloud provider.)
```

Comment:
so, it doesn't matter what CA you use here, since we explicitly tell the aggregator which CA to use (the `caBundle` field).  Using the core CA is "easy" because it's already there in most setups.